lib/utils/cli.go

```diff
@@ -125,6 +125,19 @@ func FatalError(err error) {
 	os.Exit(1)
 }
 
+// PredicateError is similar to FatalError but also prints out doc link
+// for predicate related errors.
+func PredicateError(err error) error {
```

Comment:
I'd like for this to be called `FatalPredicateError` or something like that so we don't accidentally use this in a server somewhere. 😬